docker-compose.prod.yml

```diff
@@ -11,6 +11,54 @@ services:
       pub1:
         ipv4_address: ${API_IPV4_ADDRESS?}
 
+# roach-cert:
+#   container_name: roach-cert
+#   hostname: roach-cert
+#   build: db/cert
+#   volumes:
+#     - certs-roach-0:/certs/roach-0
+#     - certs-client:/certs/client
+
+  roach-0:
+    image: cockroachdb/cockroach:latest
+    container_name: ${COMPOSE_PROJECT_NAME?}-roach-0
+    restart: always
+    hostname: roach-0
+    command: start-single-node --insecure --listen-addr=roach-0:${DB_PORT} 
+    # command: start-single-node --cluster-name=dp-crdb-cluster --logtostderr=WARNING --log-file-verbosity=WARNING --listen-addr=roach-0:26257 --advertise-addr=roach-0:26257 --certs-dir=/certs
+    volumes:
+      - roach-0-data:/cockroach/cockroach-data
+    #   - certs-roach-0:/certs
+    # depends_on:
+    #   - roach-cert
+    networks:
+      - backend-net
+
+  roach-lb:
```

Comment:
@igoychev It seems we'll keep `roach-lb` on prod as it's still in the `docker-compose.prod.yml`

Reply:
I thought on this a bit more and for prod I actually created a proper cockroach cluster with having roach-0 and roach-1 nodes - see d3613d3. This also fixes #148 where the haproxy healtcheck was not working anyway.